src/App.tsx

```diff
@@ -1,27 +1,97 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState('');
```

Comment:
I think it would be better if the titleError and userError states had a boolean value. After all, the point is that the error exists or not, right? In the case of your current solution, you don't even need these error states. Because you can simply check whether the title exists or not, as it is done in the handler function below.